stdlib/public/Concurrency/Task+TaskExecutor.swift

```diff
@@ -128,12 +128,45 @@ import Swift
 /// - Throws: if the operation closure throws
 /// - SeeAlso: ``TaskExecutor``
 @_unavailableInEmbedded
-@available(SwiftStdlib 9999, *)
+@available(SwiftStdlib 6.0, *)
+public func withTaskExecutorPreference<T, Failure>(
+  _ taskExecutor: (any TaskExecutor)?,
+  isolatedTo isolation: isolated (any Actor)? = #isolation,
```

Comment:
```suggestion
  isolation actor: isolated (any Actor)? = #isolation,
```

This follows what has been done for `AsyncSequence`

```
mutating func next(isolation actor: isolated (any Actor)?) async throws(Failure) -> Element?
```